etl/__init__.py

```diff
@@ -0,0 +1 @@
+from .definitions import defs as defs
```

Comment:
The principal export should have the same name as the file.

Dagster can use any variable name, as long as it's an instance of `Definitions`.

Reply:
I have changed the export's name to `definitions`.